pkg/runtime/runtime.go

```diff
@@ -1418,6 +1422,33 @@ func (a *DaprRuntime) initLock(s components_v1alpha1.Component) error {
 	return nil
 }
 
+func (a *DaprRuntime) initWorkflow(s components_v1alpha1.Component) error {
+	// create the component
+	workflowComp, err := a.workflowsRegistry.Create(s.Spec.Type, s.Spec.Version)
+	if err != nil {
+		log.Warnf("error creating workflow component %s (%s/%s): %s", s.ObjectMeta.Name, s.Spec.Type, s.Spec.Version, err)
```

Comment:
I'm still relatively new to Go, but I typically see `%v` used for errors rather than `%s`.
```suggestion
		log.Warnf("error creating workflow component %s (%s/%s): %v", s.ObjectMeta.Name, s.Spec.Type, s.Spec.Version, err)
```